Nightly test fixes

eloycoto opened the pull request:
- Upgrades from cilium v1.0 and v1.1 to master. 
- Delete kubernetes examples  because is already tested in all test.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cilium/cilium/5858)
<!-- Reviewable:end -->

eloycoto committed:
Test: Fix failing upgrade test from 1.0 and 1.1 versions.

Fix some pods that are not ready and deleted etcd-operator because was
not recover correctly and quorum was lost.

Fix nightly builds errors

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

aanm commented on test/k8sT/Updates.go:
```diff
@@ -127,6 +127,11 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldVersion, newV
 
+		// Delete etcd operator because sometimes when install from clean-state
+		// the quorum is lost.
+		err = kubectl.DeleteETCDOperator()
```
Why do we need to do this? This is already executed in the BeforeAll.

EDIT: It's because of the nightly, then remove this line from the BeforeAll

aanm commented on test/k8sT/Updates.go:
```diff
@@ -150,6 +155,13 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldVersion, newV
 
+		By("Waiting etcd-operator is ready")
+		// On upgrades from v1.0 and v1.1 (Where etcd-operator is not used)
+		// etcd operator pods were not ready when upgrades. With this change we make
```
I think you can remove this comment as this happens for all cilium versions.

aanm commented on test/k8sT/Updates.go:
```diff
@@ -127,11 +127,6 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldVersion, newV
 		// from master instead of create the new ones.
 		_ = kubectl.Delete(helpers.DNSDeployment())
 
-		// Delete etcd operator because sometimes when install from clean-state
```
This change seems to be in the wrong commit.

eloycoto committed:
Test: Delete nightly jobs to use kubernetes example.

Kubernetes example is already tested in all test and no need to have a
fix test for that.

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>
